src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -2,18 +2,26 @@
 
 import com.google.gson.Gson;
 import com.training.basetest.ApiBaseTest;
-import com.training.pojos.ga.authentication.Payload;
 import com.training.pojos.ga.validation.Request;
 import com.training.utilities.RestEngine;
 import io.restassured.RestAssured;
+import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class GuestAccountTests extends ApiBaseTest {
     private final String EMAIL = "email@email.com";
+    private final String PASSWORD = "password1";
+    private final String EMAIL_VALIDATION_URL = "/en/al/web/v3/guestAccounts/validation";
     private static final String REQUEST_UNSUCCESSFUL = "Request Unsuccessful";
     private static final String ERROR_PRESENT = "Error present in response";
+    private static final String RESPONSE_ERROR = "Response not as expected";
+    private static final String ERROR_CODE = "Wrong Error Code";
+    private static final String ERRORS_MESSAGE="Wrong Error Message";
+    private static final String DEVELOPERS_MESSAGE = "Wrong Developer Message";
+    private static final String INTERNAL_MESSAGE = "Wrong Internal Message";
+    com.training.pojos.ga.validation.Request request = new Request(EMAIL);
```

Comment:
lets remove this global variable, since the request would be dependent on the individual tests